extensions/amp-social-share/0.2/social-share.css.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const BASE_STYLE = {
+  'backgroundRepeat': 'no-repeat',
+  'backgroundPosition': 'center',
+  'textDecoration': 'none',
+  'cursor': 'pointer',
+  'position': 'relative',
+};
+
+/* Twitter Styling */
+export const SOCIAL_SHARE_TWITTER = {'backgroundColor': '1da1f2'};
```

Comment:
We can abbreviate naming to just `TWITTER`, etc. THe "SOCIAL_SHARE" is already represented in the `social-share.css.js` file name.